tests/symbolicator/test_unreal_full.py

```diff
@@ -82,7 +83,12 @@ def unreal_crash_test_impl(self, filename):
         self.insta_snapshot(
             {
                 "contexts": event.data.get("contexts"),
-                "exception": event.data.get("exception"),
+                "exception": {
+                    "values": [
+                        normalize_exception(x)
+                        for x in get_path(event.data, "exception", "values") or ()
```

Comment:
This is super nitpicky so do with it what you like:

- `get_path` has a `default=` keyword argument which is more natural to use in python compared to `or ...` (you should start to recognise this by now ;) ).

- The tuple type should be more thought of as a "record" type rather than an iterator.  So if you need an empty iterator using `[]` is much more semantic.

- Normally you're told not to use mutable arguments (`default=[]` in this case), however this is ok because:
  - It is not evaluated at import time but every time the list comprehension is constructed.
  - It is never modified, but rather thrown away immediately.

Reply:
I appreciate this feedback! However, I was copying this patterns from the `init.py` file, so I will rather keep this piece of code consistent with the rest in this directory.